mm2src/coins/utxo/slp.rs

```diff
@@ -1490,6 +1491,8 @@ impl From<SlpFeeDetails> for TxFeeDetails {
 impl MmCoin for SlpToken {
     fn is_asset_chain(&self) -> bool { false }
 
+    fn get_raw_transaction(&self, _req: RawTransactionRequest) -> RawTransactionFut { unimplemented!() }
```

Comment:
SLP is a custom protocol working on top of BCH (UTXO coin). You can use `utxo_common::get_raw_transaction`.